docs/input.md

```diff
@@ -90,20 +90,199 @@ def setup(): # Only drawn once, so just using setup()
 
 which results in:
 
-![Write Name](img/write_name.png)
+![Write Name](img/write_name.png){: loading=lazy }
 
 
-*Ask the user for a radius. We need to convert the string to an integer, we can do this using int(). Draw a circle of that radius at (100, 100):*
+*Ask the user for a diameter. We need to convert the string to an integer, we can do this using int(). Draw a circle of that diameter at (100, 100):*
 
-```python
+```python hl_lines="5"
 %%ignite
 
 def setup(): # Only drawn once, so just using setup()
     size(200, 200)
-    radius = int(input("What radius should the circle be?: ")) # cast the string to an integer
-    fill_circle(100, 100 radius)
+    diameter = int(input("What diameter should the circle be?: ")) # cast the string to an integer
+    fill_circle(100, 100 diameter)
 ```
 
 which results in:
 
-![Ask circle radius](img/circle_radius.png)
+![Ask circle diameter](img/circle_radius.png){: loading=lazy }
+
+### Check for key presses
+
+Spark features multiple ways to check for user key presses:
+
+| Name                            | Type                  | Description                                                  |
+|---------------------------------|-----------------------|--------------------------------------------------------------|
+| [key](#key)                     | Variable              | A string of what key was last pressed                        |
+| [keys_held()](#keys_held)       | Function              | A function that returns True if the provided key is held     |
+| [key_pressed()](#key_pressed)   | User defined function | A definable function that activates when any key is pressed  | 
+| [key_released()](#key_released) | User defined function | A definable function that activates when any key is released |
+| [key_repeated()](#key_repeated) | User defined function | A definable function that activates when any key is held     |
+
+#### key
+
+A string that is the last key pressed
+
+##### Notes
+- This variable **does not clear**, meaning if a key is pressed it will remain the value until a **new key** is pressed
```

Comment:
As above, value remains until another event is triggered. The word I'd personally use is "actuated", but that may be overly technical

Reply:
Yeah, that is somewhat my worry. Functionally it remains more or less the same in the mind of most students so I think I will keep it at pressed to avoid any possible confusion.